suite-native/module-send/src/components/OutputsReviewFooter.tsx

```diff
@@ -145,6 +172,27 @@ export const OutputsReviewFooter = ({
             }
 
             setTxid(sentTxid);
+        } else if (isRejected(sendResponse)) {
```

Comment:
nit: `else if` is not needed. There is no other unhandled option. So you can call return inside of the `if` and unwrap this code block.

Reply:
d05da19b76